frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -288,9 +305,16 @@ public void shutdown() {
 
     void setState(WorkerState newState) {
         listener.notifyChangeState(model.getModelName(), newState);
+        long timeTaken = System.currentTimeMillis() - this.startTime;
         if (state != WorkerState.WORKER_TERMINATED) {
             this.state = newState;
         }
+
+        if (state == WorkerState.WORKER_MODEL_LOADED) {
+            workerLoadTime.setValue(Long.toString(timeTaken));
+            workerLoadTime.setTimestamp(Long.toString(System.currentTimeMillis() / 1000));
```

Comment:
If the objective of the ```time/1000``` operation here is to convert to seconds, then you can use something like this : long seconds = TimeUnit.MILLISECONDS.toSeconds(millis); 

It does the same thing, but makes it easier to read the code.

Reply:
Sure. I agree. I can make this change.